components/CreditsTable.tsx

```diff
@@ -82,7 +82,9 @@ export const CreditsTable = ({
                         </h1>
                         <ul className="flex flex-col justify-center gap-4 md:justify-start">
                             {cast.map((p) => (
-                                <li key={p.id}>
+                                <li
+                                    key={`${p.id}_${Math.round(Math.random() * 1000)}`}
```

Comment:
[I think this is bad practice](https://react.dev/learn/rendering-lists#where-to-get-your-key:~:text=Keys%20must%20not%20change%20or%20that%20defeats%20their%20purpose!%20Don%E2%80%99t%20generate%20them%20while%20rendering.)